### PR TITLE
If data quality is variable individual data points should have quality flags

### LINKED ISSUE #87: Flags for data quality
Karthik Ram (#83):

> Trusting someone else's data is often very hard for downstream consumers (There is some interesting discussion in Zimmerman 2008). So if people can provide additional flags or indicators about the data quality (not QA), that could help lower the barrier to reuse. Obviously this is suggestion may not apply to certain types of data. 
> 
> Zimmerman, A.S. (2008). New Knowledge from Old Data: The Role of Standards in the Sharing and Reuse of Ecological Data. Science, Technology & Human Values, 33, 631–652.

### PR DESCRIPTION
**Pull request by ethanwhite**
Fixes #87.

### CODE REVIEW

**Comment by embaldridge** [comment]
The flow on this seems off to me.  Maybe it would be better if this were split into two sentences?  I was also wondering if we need to explain what a flag is to the novice user.

**Comment by ethanwhite** [comment]
Good call. I split it into two sentences (requiring making that sentence a set of bullets) and changed the language to avoid the jargon.

**Comment by sarahsupp** [comment]
I think the new line about the data quality of individual data points read awkwardly, but I like the addition of this information.

**Comment by ethanwhite** [comment]
I've given it two tries now. Does anyone else want to recommend an alternative?

**Comment by sarahsupp** [comment]
What if it is a sub-bullet point that simply reads:
-  warnings about known problems or inconsistencies in the data
  -  e.g., columns in the tables that to indicate the data quality of individual data points

The part that was awkward to me is the repetition of "data quality" and "individual data points" twice in the sentence.

**Comment by ethanwhite** [comment]
Nice! I didn't like having the sub-bullet so I just combined the two into a single bullet. See if this works.

File: data_sharing_ms.md [hunk 1]
```diff
@@ -79,10 +79,14 @@ Without clear metadata, datasets can be overlooked or not used due to the diffic
...
-2) how to find and access the data,
-3) suggestions on the suitability of the data for answering specific questions,
-4) warnings about known problems or inconsistencies in the data, and 5) information to check that the data are properly imported, such as the number of rows and columns in the dataset and the total sum of numerical columns [@michener1997; @zimmerman2003; @strasser2012].
+Good metadata should provide the following information [@michener1997; @zimmerman2003; @strasser2012]:
+
+* The what, when, where, and how of data collection.
+* How to find and access the data.
+* Suggestions on the suitability of the data for answering specific questions.
+* Warnings about known problems or inconsistencies in the data.
+In cases where the data quality of individual data points is variable this may include columns in the tables that to indicate the data quality of individual data points.
```
Comment (embaldridge) [inline data_sharing_ms.md]: -  If the quality of individual data points is variable, a column can be added to the table that indicates the quality of each datum.

Maybe something like this?

**Comment by ethanwhite** [comment]
Looks like I flubbed a push earlier. Take a look at it now.

**Comment by embaldridge** [comment]
Yup, that reads well now.

**Comment by ethanwhite** [comment]
Merging. If it's not quite right yet feel free to submit a follow up pull request.

**Pull request merged**